nh_observations/notifications.py

```diff
@@ -129,6 +129,11 @@ def complete(self, cr, uid, activity_id, context=None):
         }
     ]
 
+    def set_form_description_frequencies(self, available_frequencies):
```

Comment:
I'm not entirely why nh_observation has the Review Frequency defined anyways as this is currently an EWS only notification